core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -558,9 +561,12 @@ Map<String,Set<Text>> getLocalityGroups(String tableName)
    * @throws AccumuloException if a general error occurs
    * @throws AccumuloSecurityException if the user does not have permission
    * @throws TableNotFoundException if the table does not exist
+   * @throws InvalidTabletHostingRequestException if tablet hosting is requested for table with
+   *         NEVER goal
    */
   Set<Range> splitRangeByTablets(String tableName, Range range, int maxSplits)
-      throws AccumuloException, AccumuloSecurityException, TableNotFoundException;
+      throws AccumuloException, AccumuloSecurityException, TableNotFoundException,
```

Comment:
Looking into the implementation of this it only needed tablet extents and did not need tablet locations (it was ignoring them).  I updated it to call a new method in the tablet cache that allows getting the extents w/ or w/o a location. So it should not throw this exception anymore.  The pr w/ the update is #3330

Reply:
I suppressed this exception in 8694d09 as, in theory, it shouldn't happen.